packages/office-addin-manifest/.vscode/launch.json

```diff
@@ -38,6 +38,21 @@
         "${workspaceFolder}/../../../office-addin-validator/manifest-to-test/valid_excel.xml"
       ]
     },
+    {
+      "name": "Command Line: export",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceFolder}/lib/cli.js",
+      "args": [
+        "export",
+        "-o",
+        "${workspaceFolder}\\test\\out.zip",
```

Comment:
Not really in this line, but should we also add this `out.zip` to this PR?
This is more so git doesn't complain every time this command runs. We can also add this file to the `.gitignore`.

Reply:
Why would we add the out file?  This is providing an argument for the "output" argument of the command so it can be debugged . . . meaning it's the file that gets generated for debugging.

Reply:
You are right, we don't need to add this file to git.
But we should add it to `.gitignore` right?

Reply:
Not sure.  I put it there to get it to work, but I'm not sure I like the idea of the left over artifact from debugging.  Maybe it belongs in the temp directory instead.

Reply:
Yeah, I agree we should not have leftovers pushed into git. Putting into temp folder is fine as well